src/Symfony/Component/DependencyInjection/Loader/YamlFileLoader.php

```diff
@@ -532,6 +534,13 @@ private function parseDefinition($id, $service, $file, array $defaults)
                 }
 
                 $bindings = array_merge($bindings, $this->resolveServices($service['bind'], $file));
+                $bindingType = $this->isLoadingInstanceof ? BoundArgument::INSTANCEOF_BIND : BoundArgument::SERVICE_BIND;
+                foreach ($bindings as $argument => $value) {
+                    if ($value instanceof BoundArgument) {
+                        continue;
```

Comment:
no need for the continue here, let's move the next line inside the "if"

Reply:
Done